src/components/Map.js

```diff
@@ -12,13 +12,16 @@ class Map extends React.Component {
     constructor(props) {
         super(props);
 
-        let parkingSpotCoords = [];
+        let parkingSpots = [];
 
         getParkingSpots().then((response) => {
             if(response.data) {
                 const records = response.data.records;
                 for(let record of records) {
-                    parkingSpotCoords.push(record.geometry.coordinates);
+                    parkingSpots.push({
+                        coords: record.geometry.coordinates,
```

Comment:
i think this logic could be done in the service method already no ? 
The logic that you want to keep in the components is only the display logic, Anything else goes in a service to make it easier to read ;)